src/modules/firebase/services/firebaseService.ts

```diff
@@ -0,0 +1,28 @@
+import firebase from 'firebase/app';
+import 'firebase/auth';
```

Comment:
Not a necessity but we also have functional versions of these services on other projects now, if we ever want to update this to work as a hook.